plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1763,7 +1763,10 @@ action.procedure.match_dependencies=<html>Only procedures that match dependencie
 action.procedure.missing_dependencies=<small> - missing dependencies
 action.eap_loading.text=<html><b>The version you are using is EAP snapshot!</b><br><br>Before opening any workspace in this version, make a backup of the workspace.\
   <br>This version is not intended for production use and can contain critical bugs!
-action.eap_loading.title=Snapshot version
+action.eap_loading.title=Snapshot Version
```

Comment:
I would remove this message and instead test it a bit more. We have snapshots with warning they are preview anyway